tests/bluetooth/df/connection_cte_req/src/test_cte_req_enable.c

```diff
@@ -90,7 +96,34 @@ void test_set_conn_cte_req_enable_invalid_conn_handle(void)
 		      "Unexpected error value for CTE request enable with wrong conn handle");
 }
 
-void test_set_conn_cte_req_enable_before_set_rx_params(void)
+static void cte_req_params_set(void)
+{
+	g_data.cte_request_interval = REQUEST_INTERVAL_OK;
+	g_data.requested_cte_length = BT_HCI_LE_CTE_LEN_MAX;
+	g_data.requested_cte_type = BT_HCI_LE_AOD_CTE_2US;
+}
+
+static void connection_setup(void *arg)
+{
+	ARG_UNUSED(arg);
+
+	cte_req_params_set();
+
+	g_conn_handle = ut_bt_create_connection();
+}
+
+static void connection_teardown(void *arg)
+{
+	ARG_UNUSED(arg);
+
+	ut_bt_destroy_connection(g_conn_handle);
+}
+
+ZTEST_SUITE(test_hci_set_conn_cte_rx_params__before_rx_params, NULL, NULL, connection_setup,
+	    connection_teardown, NULL);
```

Comment:
These functions are called `_setup` and `_teardown` but are placed in the `_before` and `_after` arguments. I think they should be `before`/`after`